app/Http/Controllers/BoardsController.php

```diff
@@ -88,7 +88,8 @@ public function store(StoreBoard $request)
     public function show($id)
     {
         $board = Board::findOrFail($id);
-        return view('idea', compact('board'));
+        $ideas = $board->idea;
+        return view('idea', ['board'=>$board,'ideas'=>$ideas]);
```

Comment:
You can also use `$board->ideas` in the view and achieve the same result without using `$ideas` as a variable here.